src/index.tsx

```diff
@@ -60,6 +64,8 @@ export {
   PaddedContent,
   Pagination,
   RadioButtonGroup,
+  BaseRadioButtonGroup,
```

Comment:
We should export with a different name. This can make consumers have the false notion that this component must be extended and is not for standalone use. e.g.
```suggestion
  BaseRadioButtonGroup as RadioButtonGroupV2,
```

Reply:
V2 is not quite right here since it's not a replacement for the styled one. WDYT about "EmptyRadioButtonGroup"?

Reply:
Alternatively, we repurpose the old name, call the old control "HorizontalRadioButtonGroup", make it API-compatible with the old thing, and folks have a very easy breaking change to take later on.

Reply:
I went with Empty meanwhile